portal-tools/delete_stale_records.py

```diff
@@ -0,0 +1,247 @@
+"""
+APEL - DELETE stale records from the DB
+
+This script connects to a MySQL database and deletes stale records from a specified table.
+Records are only deleted if the number of recent entries (within a user-defined timeframe)
+meets or exceeds a threshold.
+
+The script supports `--dry-run` mode.
+
+NOTE:
+    Before running this script, ensure the following:
+
+    - A valid configuration file (e.g. delete_stale_records.cfg) is present and accessible.
+    - A log file path (e.g. delete_stale_records.log) is defined either in the config
+      or via the --log_config argument.
+    - The directory for the log file exists and is writable.
+
+    You can override the default paths using:
+        --db /path/to/delete_stale_records.cfg
+        --log_config /path/to/delete_stale_records.log
+
+Usage:
+    python delete_stale_records.py
+    python delete_stale_records.py --dry-run
+    python delete_stale_records.py --db /path/to/delete_stale_records.cfg --log_config /path/to/delete_stale_records.log
+"""
+
+# Requirements:
+# mysqlclient==2.1.1  # Latest package; works with Python 3.9+. Dropped support for Python 3.6.
```

Comment:
The comment is confusing. You're actually referring to the latest package available in PyPI, not the one listed here.

Just state why this version is the requirement, don't refer to something completely different that we're not using here.